Updated and New Device Drivers for graphene experiments

Evan-Walsh opened the pull request:
Updated driver for:
-Agilent 53131A (Counter)
New drivers for:
-Keithley 195A
-Agilent N9020A (Spectrum Analyzer)
- SRS 865 (Lock-In Amplifier)

caryan commented:
@Evan-Walsh a few requests for the Agilent N9020A

1. follow a more conventional naming scheme for the properties.  E.g  I expect to set/get the `sa.centreFreq` or `sa.center_freq` but not `sa.SAFreqCenter`
1. comment out properties that don't have setters/getters implemented
1. implement setters/getters for the properties rather than having differently named methods
1. what are the `QuerySCPI` and `WriteSCPI`  method for? By subclassing `deviceDrivers.lib.GPIBorEthernet` you already have methods. See e.g. [here](https://github.com/BBN-Q/Qlab/blob/develop/common/%2BdeviceDrivers/%2Blib/GPIBorEthernet.m#L79)

Evan-Walsh commented:
@caryan 
I've added a new commit to this branch:
1. I ended up merging the two codes Jess and I had for the Keithley 2400 so that I could keep my other programs unchanged
2. I've also added a driver for the Signal Recovery 7124 Lock In

caryan commented:
@Evan-Walsh I don't want to hold this out here because I'm cranky about the Agilent N9020A driver. Let's merge this and make a new issue to clean that driver up.
